plugins/community/stephenh-ts-proto/v1.176.0/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1.7
+FROM node:20.13.1-alpine3.19 AS build
+WORKDIR /app
+COPY --link package*.json .
+RUN npm ci
+RUN sed -i -e 's|/usr/bin/env node|/nodejs/bin/node|g' /app/node_modules/ts-proto/protoc-gen-ts_proto \
+ && rm -f /app/node_modules/dprint-node/*darwin*.node /app/node_modules/dprint-node/*win32*.node
```

Comment:
Partial cleanup of https://github.com/bufbuild/plugins/pull/1118#issuecomment-2000279316. If we want to get more sophisticated we can remove all versions except the one we're building for (`linux-arm64-gnu` or `linux-x64-gnu`), however just starting with removing darwin/windows.